src/utils/azureUtils.ts

```diff
@@ -34,7 +34,7 @@ export function createGroupConfigFromResource(resource: AppResource, subscriptio
             label: resource.azExtResourceType ? azExtDisplayInfo[resource.azExtResourceType ?? '']?.displayName ?? unknown : unknown,
             id: `${subscriptionId}/${resource.azExtResourceType}`,
             iconPath,
-            contextValuesToAdd: ['azureResourceTypeGroup', ...(resource.azExtResourceType ? [resource.azExtResourceType] : [])]
+            contextValuesToAdd: ['azureResourceTypeGroup', ...(resource.azExtResourceType ? [resource.azExtResourceType] : []), ...(resource.azExtResourceType ? [legacyTypeMap[resource.azExtResourceType] ?? ''] : [])]
```

Comment:
As I mentioned to Matt, I don't think you need to do the legacy stuff in here because all of our Azure Extension groups are already created. Technically, this is only for resource types that aren't part of our extension list and to have the proper id to compare it to the existing `GroupTreeItem`.

I only bring it up because this ternary is particularly nasty looking and this is a temporary workaround until we ship all of the extensions with the updated context values.

Reply:
Thanks, wasn't sure I needed it here.

> this ternary is particularly nasty looking

Ya I wish there was an easier way to conditionally add an item to an array inline. Maybe we make a util for this lol